### PR TITLE
Ability to convert a single QTI file to Learnosity JSON (WBL-81)

### PR DESCRIPTION
**Pull request by rakeshqdsa**
Can we add support for a new command line argument, --single-item yes, and if passed we assume the --input argument points to a single XML file only. Not a folder that contains many XML files and an imsmanifest.xml.

### CODE REVIEW

File: src/Services/ConvertToLearnosityService.php [hunk 1]
```diff
@@ -140,9 +144,26 @@ public function process()
         FileSystemHelper::createDirIfNotExists($this->outputPath . DIRECTORY_SEPARATOR . $this->rawPath);
 
         $this->assetsFixer = new AssetsFixer($this->organisationId);
-
-        $result = $this->parseContentPackage();
-
+        if ($this->isSingleItemConvert == 'Y' || $this->isSingleItemConvert == 'YES') {
+            $finder = new Finder();
+            $finder->files()->in($this->inputPath);
```
Comment (michaelsharman) [inline src/Services/ConvertToLearnosityService.php]: @rakeshqdsa the idea was that you pass a direct file system path to a single file, not a directory that might contain a file.

So, I might pass this:

```
./bin/mo convert:to:learnosity --input ~/Desktop/my-qti.xml --output ~/Desktop/_output --single-item yes
```

Comment (rakeshqdsa) [reply src/Services/ConvertToLearnosityService.php]: Done

**Comment by ttton** [comment]
@rakeshqdsa can you please rebase and resolve the conflict in this PR please?

**Comment by rakeshqdsa** [comment]
Done

**Pull request merged**